### DISCUSSION TITLE
Usage of shared variables in file mounts

### DISCUSSION THREAD

**Discussion by freezy** [coollabsio/coolify#3638] [Improvement Requests]
It would be really awesome if shared variables would resolve when defining content for file mounts. It would allow for a much more dynamic configuration, since file mounts are often used for configuration.

It would also be great if the documentation would be more precise where exactly the shared variables are resolved. The place I've found so far are environments. ~I haven't tried container labels yet.~ I did now, and they don't work there either. Neither in ports exposes, and I assume port mappings.

Thanks!

**Comment by luishugoooo** [comment]
I am stumbling on the same issue currently, cannot use env vars to do directory bind mounts.
